### DISCUSSION TITLE
What do the red highlighted names mean?

### DISCUSSION THREAD

**Discussion by appleb4** [AAndyProgram/SCrawler#77] [Q&A] [answered]
First off, thanks for this amazing program. Very helpful

I'm following some Tiktok accounts with autodownloader running, some of them have highlighted names though and I'm not sure why. It returns to normal when I rightclick and download data, but this happens even without any missing posts.

**Comment by AAndyProgram** [comment, marked as answer]
Red means that the user's profile has been deleted. Sorry, forgot to add this to the wiki. I [added](https://github.com/AAndyProgram/SCrawler/wiki#user-colors) it right now.
If your profiles are marked red by mistake, please post them and I'll try to fix this problem.
P.S. And thank you for using SCrawler and for such kind words! 😊

**Reply by appleb4** [reply]
Thanks for the explanation. I actually haven't seen it happen since I posted this, maybe the newest version fixed it or it was something on my end.

**Reply by AAndyProgram** [reply]
Okay then. "Deleted" users will be skipped in AutoDownloader, but they can still be downloaded manually (select users and click download).

**Reply by appleb4** [reply]
It happened again and it is on my end (or TikTok's), when it sometimes fails to load in my web browser, it brings me to a generic TikTok error page saying something went wrong with a button to retry, which usually works first or second try. SCrawler tried auto-downloading during this time, failed to connect and considers the users deleted. I'm not entirely sure how to recreate it, seems random.

**Reply by AAndyProgram** [reply]
>I'm not entirely sure how to recreate it, seems random.

What do you want to recreate? "Deleted" users will be skipped in AutoDownloader, but **they can still be downloaded manually** (select users and click download).

>it is on my end

In this case, I can't fix it. I can only suggest that you use a more stable internet connection and/or use a VPN. I don't know all the errors that can be returned with a response because TikTok doesn't have proper API documentation. Currently, a TikTok user is considered deleted when the response returns a 'ConnectionClosed' error.

https://github.com/AAndyProgram/SCrawler/blob/7d169acebc91ad777dddea9b628936ec5a199748/SCrawler/API/TikTok/UserData.vb#L85-L86

----
**If you like SCrawler, please like the program on [this site](https://alternativeto.net/software/scrawler/about/): https://alternativeto.net/software/scrawler/about/.**

**Reply by appleb4** [reply]
I meant that it's not so much my connection but TikTok refusing it every once in awhile but I guess there's not much to be done about that. Thanks for the reply

**Reply by AAndyProgram** [reply]
You are welcome 😉
